lucky-number-model/Models/LuckyNumber.cs

```diff
@@ -17,5 +20,36 @@ public class LuckyNumber
         public decimal Balance { get; set; } // the balance for this game
 
         public Boolean isWinner { get; set; } // a flag for when the lucky number matches the spin
+
+        public String Message
+        {
+            get
+            {
+                if (Balance <= 0) { return "GAME OVER: Spin to try again"; }
+                return "";
+            }
+
+            set
+            {
+
+            }
+        }
+        public int [] spinner
+        {
+            get
+            {
+                Random randint = new System.Random();
+                int[] spinner = new int[3];
```

Comment:
By declaring this as int[] spinner, you are creating a local variable inside your getter, rather than using the instance variable _spinner. Instead, remove line 42, and use _spinner[0] in line 43 and following.